templates/common/_base/files/configure-ovs-network.yaml

```diff
@@ -490,6 +490,63 @@ contents:
       echo ""
     }
 
+    get_ip_from_ip_hint_file() {
+      local ip_hint_file="$1"
+      if [[ ! -f "${ip_hint_file}" ]]; then
+        return
+      fi
+      ip_hint=$(cat "${ip_hint_file}")
+      echo ${ip_hint}
+    }
+
+    try_to_bind_address() {
```

Comment:
I will add comment for it.
nodeip-configuration checks it as currently node-ip is running in parallel/after ovs-configuration and this issue is caused by ovs-configuration

Reply:
My idea was to change it only for new logic for now. I am not sure this issue happens on any other platform rather than metal and was not sure regarding pushing it for all. 
So the change would be relevant only in nodeip-configuration is running before ovs and if not just do nothing.
I assume after this will be pushed we can remove this workaround from node-ip

Reply:
Why is this issue considered to be caused by configure-ovs? AFAIK this would happen without configure-ovs as well. For example, if openshift-sdn supported IPv6 we would have the same issue, and openshift-sdn does not use configure-ovs at all. I might be missing something?
As far as workarounds go, I am much more comfortable, if we are considering it only for when nodeip-configuration runs, that the workaround is in nodeip-configuration, as I think we have much more control over it there, being a go program and this being bash script that is giving us a lot of trouble each time we add functionality to it.

Reply:
I debugged it a lot and problem happens right after ovs-configuration, we can't leave this logic in nodeip-configuration as it will not help. This is the reason i moved it here. 
What is happening is that network not bindable right after ovs-configuration on ipv6 and crio fails. 
My another idea was to fix it actually in crio service by verifying same logic but not sure if it is nicer. @cybertron ?
If we leave it here and not moving it to crio we need to decide if we want to run it all the time or just with new logic. As for start i would prefer running only in case nodeip was running but i can change it to run all the time if your prefer

Reply:
I am trying to say that i am not sure that ovs-config causes the issue but cause of new dependency we must verify it here or in crio as nodeip will bind address with no problem (tested dozen of times already)

Reply:
Ah, I get it know. nodeip would bind with the original interface, but we would need to retest again after br-ex is configured.
Sorry for the confusion.

Reply:
yep, so we are ok here?

Reply:
I assume now the question is if we should do it all the time or only if node ip is running? @cybertron @jcaamano wdyt?

Reply:
IMO, if this is to avoid tentative phase of IPv6, we have seen this before and it would help if we generally do it while the problem is fixed on NM side somehow.